metric-example/metric-expressions-by-example.md

```diff
@@ -205,22 +155,20 @@ Now we have host-based dimension tuples for the top PGIs and want to fill in the
 another metric `builtin:host.mem.avail.bytes`. A general pattern to apply topping with `metricA`
 but show the values from `metricB` for the topped dimensions is:
 ```
-(<metricA>:sort(...):limit(...))*(0)+(<metricB>)
+<metricA>:sort(...):limit(...) * 0 + <metricB>
 ```
 
 Let's put it all together:
 ```
-(
-  builtin:tech.generic.mem.usage
+builtin:tech.generic.mem.usage
   : sort(value(max,descending))
```

Comment:
We use a bit more indentation here than previously, e.g. here there is no white-space before the colons:
```
temprerature
: auto
: toUnit(Fahrenheit,Kelvin)
: filter(
    series(max, gt(1.9))
  )
```

Both styles are fine for me, but we should only use one of them.